Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -0,0 +1,350 @@
+
+      // Wiimote
+      if (Settings().IsBluetoothPassthroughEnabled())
+      {
+        const auto ios = IOS::HLE::GetIOS();
+        auto device = ios ? ios->GetDeviceByName("/dev/usb/oh1/57e/305") : nullptr;
+
+        if (device != nullptr)
+          std::static_pointer_cast<IOS::HLE::Device::BluetoothBase>(device)->UpdateSyncButtonState(
+              IsHotkey(HK_TRIGGER_SYNC_BUTTON, true));
+      }
+      // TODO Debugging shortcuts (Separate PR)
+      Common::SleepCurrentThread(1000 / 60);
+
+      if (Settings().IsWiiGameRunning())
+      {
+        int wiimote_id = -1;
+        if (IsHotkey(HK_WIIMOTE1_CONNECT))
+          wiimote_id = 0;
+        if (IsHotkey(HK_WIIMOTE2_CONNECT))
+          wiimote_id = 1;
+        if (IsHotkey(HK_WIIMOTE3_CONNECT))
+          wiimote_id = 2;
+        if (IsHotkey(HK_WIIMOTE4_CONNECT))
+          wiimote_id = 3;
+        if (IsHotkey(HK_BALANCEBOARD_CONNECT))
+          wiimote_id = 4;
+
+        // TODO Implement Wiimote connecting / disconnecting (Separate PR)
+        // if (wiimote_id > -1)
+      }
+
+      // Graphics
+      if (IsHotkey(HK_INCREASE_IR))
+        ++g_Config.iEFBScale;
+      if (IsHotkey(HK_DECREASE_IR))
+        g_Config.iEFBScale = std::max(g_Config.iEFBScale - 1, static_cast<int>(SCALE_AUTO));
+      if (IsHotkey(HK_TOGGLE_CROP))
+        g_Config.bCrop = !g_Config.bCrop;
+      if (IsHotkey(HK_TOGGLE_AR))
+        g_Config.iAspectRatio = (g_Config.iAspectRatio + 1) & 3;
+      if (IsHotkey(HK_TOGGLE_EFBCOPIES))
+        g_Config.bSkipEFBCopyToRam = !g_Config.bSkipEFBCopyToRam;
+      if (IsHotkey(HK_TOGGLE_FOG))
+        g_Config.bDisableFog = !g_Config.bDisableFog;
+      if (IsHotkey(HK_TOGGLE_DUMPTEXTURES))
+        g_Config.bDumpTextures = !g_Config.bDumpTextures;
+      if (IsHotkey(HK_TOGGLE_TEXTURES))
+        g_Config.bHiresTextures = !g_Config.bHiresTextures;
+
+      Core::SetIsThrottlerTempDisabled(IsHotkey(HK_TOGGLE_THROTTLE, true));
+
+      if (IsHotkey(HK_DECREASE_EMULATION_SPEED))
+      {
+        auto speed = Settings().GetEmulationSpeed() - 0.1;
+        speed = (speed <= 0 || (speed >= 0.95 && speed <= 1.05)) ? 1.0 : speed;
+        Settings().SetEmulationSpeed(speed);
+      }
+
+      if (IsHotkey(HK_INCREASE_EMULATION_SPEED))
+      {
+        auto speed = Settings().GetEmulationSpeed() + 0.1;
+        speed = (speed >= 0.95 && speed <= 1.05) ? 1.0 : speed;
+        Settings().SetEmulationSpeed(speed);
+      }
+
+      // Slot Saving / Loading
+      if (IsHotkey(HK_SAVE_STATE_SLOT_SELECTED))
+        m_parent->StateSaveSlot();
+
+      if (IsHotkey(HK_LOAD_STATE_SLOT_SELECTED))
+        m_parent->StateLoadSlot();
+
+      // Stereoscopy
+      if (IsHotkey(HK_TOGGLE_STEREO_SBS) || IsHotkey(HK_TOGGLE_STEREO_TAB))
+      {
+        if (g_Config.iStereoMode != STEREO_SBS)
+        {
+          // Disable post-processing shader, as stereoscopy itself is currently a shader
+          if (g_Config.sPostProcessingShader == "dubois")
```

Comment:
I have absolutely no idea, I just copied it from Wx where it wasn't explained either. But I guess it's the anaglyph shader(?)